### PR TITLE
Auth features

### PR DESCRIPTION
**Pull request by StormTrooper-007**
Authentication and Exception features

### CODE REVIEW

File: backend/src/main/java/com/neuefische/backend/services/TimedOutService.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    public List<TimedOut> getAllTimedOut() {
+        return timedOutRepository.findAll();
+    }
+
+    public void deleteTimedOut(String id) {
+        timedOutRepository.deleteById(id);
+    }
+
+}
```
Comment (Kajochi) [inline backend/src/main/java/com/neuefische/backend/services/TimedOutService.java:38]: Hey, in almost every new classes you forgot the last empty line ;)

File: frontend/src/pages/RegisterPage.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import {FormEvent, useState} from "react"
+import {useRegisterMutation} from "../features/api/apiSlice.tsx"
+import {useNavigate} from "react-router-dom"
+
+function RegisterPage() {
+    const [username, setUsername] = useState<string>("")
+    const [email, setEmail] = useState<string>("")
+    const [password, setPassword] = useState<string>("")
+    const [repeatPassword, setRepeatPassword] = useState<string>("")
+    const [register, {isError, isSuccess, data, error}] = useRegisterMutation()
```
Comment (Kajochi) [inline frontend/src/pages/RegisterPage.tsx:11]: What does the apiSlice.tsx do?

**Review by Kajochi — APPROVED** [review]
What a huge PR xD I don´t view everything but that which i have seen looks very good!:D Keep on  rocking!

File: frontend/src/pages/RegisterPage.tsx (hunk 2 above)
Comment (StormTrooper-007) [reply frontend/src/pages/RegisterPage.tsx:11]: it handles the frontend request logic to the backend

File: backend/src/main/java/com/neuefische/backend/services/TimedOutService.java (hunk 1 above)
Comment (StormTrooper-007) [reply backend/src/main/java/com/neuefische/backend/services/TimedOutService.java:38]: Yes thats true, Thanks alot

**Pull request merged**